### DISCUSSION TITLE
외부 개발사 자체 결제 내역 전송 API 호출시 403 오류 문의

### DISCUSSION THREAD

**Discussion by Contact-Sendman** [commerce-api-naver/commerce-solutions#53] [묻고 답하기]
자체 결제 API를 테스트 중인데요,
아래와 같은 에러를 받고 있습니다.

토큰은 SELF type으로 저희 솔루션에 대해 발급받아 이용 중이고, 
파라미터를 이리저리 바꿔봐도 같은 에러가 계속 발생합니다.

```
{
  "code" => "UNKNOWN_ERROR",
  "message" => "Unknown Error - An error occurred during operation",
  "timestamp" => "2024-11-09T23:15:51.020+09:00",
  "traceId" => "pr1-212535-7fsjdj^1730723220447^55821721"
}
```

**Comment by commerce-api-naver** [comment]
안녕하세요. 커머스API 이민석입니다.

알려주신 호출건의 HTTP 요청 헤더 상 Content-Type이 "application/octet-stream"으로 확인됩니다.
대상 API는 "application/json"으로 호출해주셔야 합니다. MIME Type 수정 후 다시 한 번 호출해보시기 바랍니다.
- 각 API에서 요구하는 요청 MIME Type은 API문서에 수록되어있습니다.

**Reply by Contact-Sendman** [reply]
덕분에 해결했습니다 감사합니다!
